test/e2e/components/vl-alert.js

```diff
@@ -41,7 +41,9 @@ class VlAlert extends VlElement {
     }
 
     async getActions() {
-        return this._innerText('#actions');
+      const alert = await this._getAlert();
```

Comment:
Niet hier geïntroduceerd, maar de functie `_getAlert` zou ik verwijderen, die returnt de Shadow DOM, maar die kan voor de leesbaarheid misschien beter rechtstreeks in de andere functies gebruikt worden? Dit zou dan worden:
```JavaScript
// before
const alert = await this._getAlert();
const slot = await alert.findElement(By.css('#actions-slot'));

// after
const slot = await this.shadowRoot.findElement(By.css('#actions-slot');
```